canvas_modules/harness/package.json

```diff
@@ -62,7 +67,10 @@
     "grunt-yamllint": "0.3.0",
     "html-webpack-plugin": "5.6.0",
     "ibm-design-icons": "2.1.4",
+    "istanbul": "0.4.5",
+    "istanbul-instrumenter-loader": "3.0.1",
```

Comment:
Is that actually used?

Reply:
I was still able to generate a coverage report without these, so I removed them